frontend/src/store/episode/actions.ts

```diff
@@ -41,6 +42,24 @@ const createComment = createAsyncThunk<Comment, CommentFormCreatePayload, AsyncT
   return comment;
 });
 
+const deleteComment = createAsyncThunk<Comment[], number, AsyncThunkConfig>
+(ActionType.DELETE_COMMENT, async (id, { extra, getState }) => {
+  const { commentApi, notificationService } = extra;
+  const { episode } = getState();
+  const comments = episode.comments.filter((comment) => comment.id !== id);
+
+  await commentApi.delete(id);
+  notificationService.success(NotificationTitle.SUCCESS, NotificationMessage.COMMENT_DELETED);
+
+  return comments;
```

Comment:
Should we notificate othe users that episode was deleted through the sockets?

Reply:
agreed!

Reply:
It looks like we still not notificate users, through the sockets about comment deletion)